cpuid.go

```diff
@@ -1146,81 +1141,82 @@ func hasSGX(available, lc bool) (rval SGXSupport) {
 	return
 }
 
-func support() (Flags, AmxFlags) {
+func support() (Flags, FlagSet) {
 	mfi := maxFunctionID()
 	vend, _ := vendorID()
 	if mfi < 0x1 {
-		return 0, 0
+		return 0, nil
 	}
-	flags := uint64(0)
-	amxFlags := AmxFlags(0)
+
+	fs := make(FlagSet, LASTID/64+1)
```

Comment:
Would it be safer to have a function for doing this, e.g. `newFlagSet()` or smth

Reply:
This is in PR #59, closing in this conversation.